database/database.json

```diff
@@ -0,0 +1,10 @@
+{
+    "dev": {
+        "driver": "pg",
+        "user": "postgres",
+        "password": "opensesame",
+        "host": "localhost",
+        "database": "cyf-tms",
+        "port": "5432"
+    }
+}
```

Comment:
Your editor is not leaving a line break at the end of the file. This is not good for unix systems. Please fix it